marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-load-reader.json

```diff
@@ -1,4 +1,4 @@
 {
-  "role-name": "data-hub-load-data-reader",
+  "role-name": "data-hub-load-reader",
```

Comment:
Shouldn't this be "data-hub-ingestion-reader" for consistency with "data-hub-mapping-reader"? I think "hub-central-load-reader" is fine since HC wants to use the term "load", but for core DH, I think we should stay with "ingestion" for consistency with existing core DH capabilities.

Reply:
I'm good with that. Makes sense to make it consistent.